### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by MathiasHobler**
please take a look

### CODE REVIEW

File: src/components/Inputfield/ImageInput.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		font-size: 0.8em;
+		text-transform: uppercase;
+		cursor: pointer;
+
+		&:hover {
+			color: white;
+		}
+	}
+
+	div {
```
Comment (JessicaLoers) [inline src/components/Inputfield/ImageInput.jsx:37]: Perhaps you will find a more semantically descriptive element 🤟

File: src/components/Form/Form.jsx [hunk 2]
```diff
@@ -39,79 +47,121 @@ const Form = () => {
...
 	}
 
+	useEffect(() => {
+		requiredData();
+		console.log(newEvent);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [newEvent]);
+
+	function requiredData() {
+		if (newEvent.title === '') {
```
Comment (JessicaLoers) [inline src/components/Form/Form.jsx:57]: Perhaps this is also a case for a switch case

**Review by JessicaLoers — COMMENTED** [review]
**Great Job** 👷‍♂️
Please note my comments.

File: src/components/Form/Form.styled.jsx [hunk 3]
```diff
@@ -1,17 +1,69 @@
...
-export default FormContainer;
+export const Container = styled.section`
+	display: flex;
+	position: absolute;
+	top: 4em;
+	flex-direction: column;
+	align-content: flex-end;
+	gap: 1em;
+	margin-top: 1em;
+`;
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Form/Form.styled.jsx:69]: Done

File: src/components/EventCard/EventCard.jsx [hunk 4]
```diff
@@ -19,11 +19,7 @@ const EventCard = ({event, deleteEvent}) => {
 					toggleDetail();
 				}}
 			>
-				<img
-					src={event.pictures.eventPicture}
-					alt="EventPicture"
-					aria-label="EventPicture"
-				></img>
+				<img src={event.eventPicture} alt="EventPicture" aria-label="EventPicture"></img>
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/EventCard/EventCard.jsx:22]: Done

File: src/components/Inputfield/Inputfield.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+				}}
+				autoFocus={false}
+				required="required"
+				rows="12"
+				cols="50"
+			></textarea>
+			<span htmlFor={label}>{label}</span>
+		</InputContainer>
+	);
+};
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Inputfield/Inputfield.jsx:43]: Done

File: src/components/Form/Form.jsx [hunk 6]
```diff
@@ -39,79 +47,121 @@ const Form = () => {
...
+				return <Success error={error} />;
+		}
+	}
+
 	return (
 		<>
+			<Bubble>
+				<div></div>
+				<div></div>
+			</Bubble>
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Form/Form.jsx:94]: This is needed at the moment to see the Glaseffekt of the form component at the moment with Iphone Se view

File: src/components/Inputfield/ImageInput.jsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+
+const ImageUpload = ({setValue}) => {
+	return (
+		<InputContainer>
+			<div>
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Inputfield/ImageInput.jsx:6]: Done

File: src/components/Inputfield/ImageInput.jsx (hunk 1 above)
Comment (MathiasHobler) [reply src/components/Inputfield/ImageInput.jsx:37]: Done

File: src/components/Inputfield/ImageInput.jsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+		color: rgba(255, 255, 255, 0.25);
+		font-size: 1em;
+		text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
+		text-transform: uppercase;
+		pointer-events: none;
+	}
+
+	input {
+		width: 100%;
+		padding: 0.625em;
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Inputfield/ImageInput.jsx:64]: Done

File: src/components/Form/Form.styled.jsx [hunk 9]
```diff
@@ -1,17 +1,69 @@
...
+		font-weight: 500;
+		line-height: 42px;
+		text-align: center;
+	}
+
+	article {
+		display: flex;
+		position: relative;
+		bottom: -13em;
+		flex-direction: row;
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Form/Form.styled.jsx:50]: Done

File: src/components/Form/Form.styled.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Form/Form.styled.jsx:69]: This is specific for the inner Form Container, font-family and color comes to global styles

File: src/components/Form/Form.jsx (hunk 2 above)
Comment (MathiasHobler) [reply src/components/Form/Form.jsx:57]: Adjusted the error message, but didn't know how to make it a switch case

File: src/components/Form/Form.jsx [hunk 10]
```diff
@@ -39,79 +47,121 @@ const Form = () => {
...
+					)}
+				</article>
 			</FormContainer>
 		</>
 	);
 };
 
 export default Form;
+
+const Bubble = styled.div`
```
[the remark this answers is not in the document]
Comment (JessicaLoers) [reply src/components/Form/Form.jsx:144]: Ahhh 🤓
In this case, I would name the component more descriptively.

File: src/components/Form/Form.styled.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (JessicaLoers) [reply src/components/Form/Form.styled.jsx:63]: Okay 👍

**Pull request merged**